lib/main_common.pm

```diff
@@ -157,8 +157,14 @@ sub load_autoyast_tests {
     else {
         loadtest("autoyast/repos");
         loadtest("autoyast/clone");
+        if (get_var('SP3ORLATER') && check_var('FILESYSTEM', 'btrfs')) {
+            loadtest "autoyast/verify_autoinst_btrfs";
+        }
         loadtest("autoyast/logs");
     }
+    if (get_var('SP3ORLATER') && check_var('FILESYSTEM', 'btrfs')) {
```

Comment:
oops, I merged it now but I guess this is never true. I have never seen this variable "SP3ORLATER". Rather use `sle_version_at_least('12-SP3') || check_var('DISTRI', 'opensuse')`